src/anti_patterns/borrow_clone.md

```diff
@@ -63,11 +60,7 @@ should be understood fully before assessing whether the clone is required or
 not.
 
 Also be sure to always run `cargo clippy` in your project, which will detect
-some cases in which `.clone()` is not necessary, like
-[1](https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone),
-[2](https://rust-lang.github.io/rust-clippy/master/index.html#clone_on_copy),
-[3](https://rust-lang.github.io/rust-clippy/master/index.html#map_clone) or
-[4](https://rust-lang.github.io/rust-clippy/master/index.html#clone_double_ref).
+some cases in which `.clone()` is not necessary.
```

Comment:
The last link is broken. I can see old mentions of `clone_double_ref`, perhaps it was removed. More importantly, clippy is unlikely to recommend removing `.clone()` if the compiler requires it (which is the premise of this page). But clippy can find something else that could be improved, which would in turn make `.clone()` unnecessary (e.g. clippy might suggest passing arguments to a function by reference so the value is not consumed). No need to focus on clone-specific lints.